sys/callout.c

```diff
@@ -90,5 +91,12 @@ void callout_process(realtime_t time) {
 
       elem = next;
     }
+    if (current_bucket == last_bucket)
+      break;
+
+    current_bucket = (current_bucket + 1) % CALLOUT_BUCKETS;
   }
+
+  /* Update ci.last. */
```

Comment:
Well... that's obvious... could you add a comment to `last` field explaining its purpose?